Src/Web/Web.Shared.Net/ApplicationInsightsHttpModule.cs

```diff
@@ -81,6 +111,17 @@ private void OnEndRequest(object sender, EventArgs eventArgs)
 
                 if (this.IsFirstRequest(httpApplication))
```

Comment:
:) I did not change that. You can talk to @madeleinefletcher if you understand what is the question